usagov/vote-gov-drupal
# VOTE-1121 Reduce unused CSS

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-1121

## Description

Removed the unused styles from the touchpoints form. This is loaded onto every page, so removing improves lighthouse performance scores across the site.

## Deployment and testing

### Post-deploy steps

1. cd into votegov, npm run build
2. lando retune

### QA/Testing instructions

1. Re-run the lighthouse performance tests for each page type, and confirm the error linked in the ticket no longer exists. Note that other errors will be fixed as part of other tickets.

## Checklist for the Developer




## Checklist for the Peer Reviewers
